test_unbounded_knapsack.py

```diff
@@ -0,0 +1,52 @@
+"""
+Created on Fri Oct 16 09:31:07 2020
+
+@author: Dr. Tobias Schröder
+@license: MIT-license
+
+This file contains the test-suite for the unbounded_knapsack problem.
+"""
+import unittest
+
+from unbounded_knapsack import unbounded_knapsack as k
+
+
+class Test(unittest.TestCase):
+    def test_base_case(self):
+        """
+        test for the base case
+        """
+        cap = 0
+        val = [0]
+        w = [0]
+        c = len(val)

```

Comment:
Use real variable names as discussed in CONTRIBUTING.md.  What is a `w` and why should the reader of this code be forced to guess if it is a wager, a weight, or a whale?

Reply:
> Use real variable names as discussed in CONTRIBUTING.md. What is a `w` and why should the reader of this code be forced to guess if it is a wager, a weight, or a whale?

OK sir, will change it and surely update with in few min.

Reply:
> Use real variable names as discussed in CONTRIBUTING.md. What is a `w` and why should the reader of this code be forced to guess if it is a wager, a weight, or a whale?

sir i have changed the name 
now, do i need to generate one more pull request?

Reply:
@cclauss sir please let me know if is there any changes that i should change.